README.md

```diff
@@ -10,7 +10,7 @@ None
 
 | Variable | Description | Default |
 |----------|-------------|---------|
-| virtual\_interface | dict holding configurations. highly OS-dependant. see the example below | {} |
+| virtual\_interfaces | array holding configurations. highly OS-dependant. see the example below | [] |
```

Comment:
although i used to write `array` in the past, `list`, which is the canonical name in ansible (or python?) is used in recent roles.